pkg/v1/providers/infrastructure-aws/v1.2.0/yttcc/overlay.yaml

```diff
@@ -100,3 +100,22 @@ spec:
       value: #@ vars[configVariable]
     #@ end
     #@ end
+    #@overlay/append
+    - name: TKR_DATA
```

Comment:
please add a comment that this should be removed (only conditionally included in test scenarios)